### ISSUE TITLE
Glance cards do not update when state changes, need manual refresh

### ISSUE THREAD

**Issue by lbschenkel** [home-assistant/home-assistant-polymer#1791]
**Home Assistant release with the issue:**

0.80.1

**Last working Home Assistant release (if known):**

0.79.x

**Browser and Operating System:**

*Clients:*
Ubuntu 18.04 and macOS 10.13.6
Firefox 62.0.3 (webcomponents enabled via `about:config`) and Chromium 69.0.3497.81

*Server:*
Hass.io containers running on Docker 18.06.1-ce on Debian 9.5/Proxmox kernel:
```
Linux pve 4.15.18-7-pve #1 SMP PVE 4.15.18-26 (Thu, 04 Oct 2018 11:03:06 +0200) x86_64 GNU/Linux
```
Machine is a Core i5 with 16 GB of RAM.

**Description of problem:**

State of entities (lights, motion sensors, etc.) does not automatically update on Lovelace UI. It does update when the page is manually refreshed. In the old UI (`/states`) and developer tools (`/dev-states`) the state *does* update without a refresh, so this issue is restricted to Lovelace.

It happens on both Chromium and Firefox, on macOS and Linux.

**Edit 1**: I noticed now that entity cards and picture-glance cards actually update, but glance cards don't. Because I had an overview panel made up entirely of glance cards, this threw me off. It looks like the issue is restricted to glance cards. Amended title.

**Javascript errors shown in the web inspector (if applicable):**

None.

**Additional information:**

No errors logged in the browser console, no errors logged in the server log.

**Comment by lbschenkel** [comment]
I just noticed that it looks like that the problem is with glance cards specifically. Updated issue:
> Edit 1: I noticed now that entity cards and picture-glance cards actually update, but glance cards don't. Because I had an overview panel made up entirely of glance cards, this threw me off. It looks like the issue is restricted to glance cards. Amended title.

**Comment by balloob** [comment]
Fixed in 0.80.2!
